src/components/common/Select/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { arrayOf, bool, func, number, oneOfType, string, shape } from 'prop-types';
+
+import './Select.scss';
+
+const Select = ({ title, placeholder, options, valid, value, onChange }) => {
+  const [display, setDisplay] = useState(false);
```

Comment:
You can omit that false and useState will be initialized with a falsy value anyway.